ci/README.md

```diff
@@ -1,5 +1,8 @@
-# Dagger
+# Go commands
+In order to use the following go commands Docker must be running. You can build them with by going into the `ci` subdirectories and running `go build ./...`. For running them you stay in the same directory and use `go run ` followed by the command name and its potential arguments.
```

Comment:
> Docker must be running

And Go must be installed on the machine. (later, we'll provide commands for full docker-contained execution)

Reply:
I'll change it thanks